include/circt/Dialect/SV/SVVisitors.h

```diff
@@ -43,10 +43,10 @@ class Visitor {
             FWriteOp, FFlushOp, SystemFunctionOp, VerbatimOp, MacroRefOp,
             FuncCallOp, FuncCallProceduralOp, ReturnOp, IncludeOp, MacroErrorOp,
             // Type declarations.
-            InterfaceOp, InterfaceSignalOp, InterfaceModportOp,
-            InterfaceInstanceOp, GetModportOp, AssignInterfaceSignalOp,
-            ReadInterfaceSignalOp, MacroDeclOp, MacroDefOp, FuncOp,
-            FuncDPIImportOp,
+            InterfaceOp, SVVerbatimSourceOp, InterfaceSignalOp,
+            InterfaceModportOp, InterfaceInstanceOp, GetModportOp,
+            AssignInterfaceSignalOp, ReadInterfaceSignalOp, MacroDeclOp,
+            MacroDefOp, FuncOp, FuncDPIImportOp,
```

Comment:
This should probably not be a type declaration. However, that relates to the earlier comment about creating a new `SV*.td` file for this to live in.